bluesky/examples.py

```diff
@@ -473,7 +473,8 @@ def stage(self):
         self._file_stem = short_uid()
         self._path_stem = os.path.join(self.save_path, self._file_stem)
```

Comment:
Oh, I see. The handler was ignoring root as well, so this "worked" as is but would not have worked with a new root. Fixed in latest commit.